onnxruntime/core/platform/path_lib.cc

```diff
@@ -17,7 +17,7 @@
 #include <PathCch.h>
 #pragma comment(lib, "PathCch.lib")
 // Desktop apps need to support back to Windows 7, so we can't use PathCch.lib as it was added in Windows 8
-#elif WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
+#elif WINVER < _WIN32_WINNT_WIN8
```

Comment:
Project is being built with WINAPI_PARTITION_DESKTOP even in UWP for the time being, because some code in flatbuffers had getenv whose definition is hidden if you build for UWP. I'm replacing WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP) with a Windows version check (since UWP is >= Win8, this also ensures the UWP build doesn't use the desktop symbol).

Reply:
And that flatbuffers code isn't called in product code so getenv isn't in the dll import list. The desktop partition is needed not to break the build but it doesn't add any desktop symbols to the binary. I'll work on some more cleanup and making all the code compilable in UWP mode after 1.5.

Reply:
@skottmckay  FYI about the flatbuffers issue.